app/src/main/java/com/github/codetanzania/ui/activity/ReportIssueActivity.java

```diff
@@ -282,15 +275,8 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     @Override
     public void onRequestPermissionsResult(
             int requestCode, @NonNull String permissions[], @NonNull int grantResults[]) {
+        mCurrentFragment.onRequestPermissionsResult(requestCode, permissions, grantResults);
         switch (requestCode) {
-            case REQUEST_ACCESS_FINE_LOCATION:
-                if (grantResults.length > 0 && grantResults[0] == PackageManager.PERMISSION_GRANTED) {
-                    // change the fragment.
-                    // LocationSelectorFragment allows user to select the location
-                    LocationSelectorFragment frag = LocationSelectorFragment.getNewInstance(null);
-                    setCurrentFragment(R.id.frl_FragmentOutlet, LOCATION_SERVICE, frag);
-                }
-                break;
             case REQUEST_ACCESS_CAMERA:

```

Comment:
This case is no longer needed. We're now using Native Camera App Provided by the Android devices